client/src/main/java/org/eclipse/hono/client/impl/CommandConsumerHandler.java

```diff
@@ -0,0 +1,107 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+
+package org.eclipse.hono.client.impl;
+
+import java.util.Objects;
+
+import org.eclipse.hono.client.CommandContext;
+
+import io.vertx.core.Handler;
+
+/**
+ * Represents a command handler used in a command consumer.
+ */
+public class CommandConsumerHandler {
```

Comment:
is this class supposed to be extended? If not, I propose to declare it final, otherwise the methods that are not supposed to be overridden should be declared final.